ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -1499,9 +1499,15 @@ impl<'a> Assemble<'a> {
             .collect::<Vec<_>>();
 
         // unwrap safe on account of linker symbol names not containing internal NULL bytes.
-        let va = symbol_to_ptr(self.m.func_decl(func_decl_idx).name())
-            .map_err(|e| CompilationError::General(e.to_string()))?;
-        self.emit_call(iidx, fty, Some(va), None, &args)
+        match self.m.func_decl(func_decl_idx).name() {
```

Comment:
Why don't we just filter these away at serialisation time?

Reply:
We *might* be able to make use of the information -- it's not yet clear to me.

In the interim, their presence in a trace prevents that trace being compiled. So this is a quick fix for that; and, perhaps, we can do something more appropriate later.